docs/tables/crowdstrike_intel_actor.md

```diff
@@ -0,0 +1,46 @@
+# Table: crowdstrike_intel_actor
+
+With the Actors API, Falcon Intel subscribers can search for data about actors that CrowdStrike is tracking.
+
+## Examples
```

Comment:
Looking at the columns, is there any interesting information in the `capability`, `entitlements`, or `group` columns? Similar to the host table, the ratio of number of columns to examples seems low